docs/theme/included/powerline.rst

```diff
@@ -79,6 +79,7 @@ The following Liquidprompt config options are respected:
 * :attr:`LP_ENABLE_SCREEN_TITLE`
 * :attr:`LP_ENABLE_SHORTEN_PATH`
 * :attr:`LP_ENABLE_SVN`
+* :attr:`LP_ENABLE_TERRAFORM`
```

Comment:
This line needs to be removed since terraform isn't in the default powerline prompt